llama_stack/providers/adapters/inference/ollama/ollama.py

```diff
@@ -63,7 +63,15 @@ async def completion(
         stream: Optional[bool] = False,
         logprobs: Optional[LogProbConfig] = None,
     ) -> AsyncGenerator:
-        raise NotImplementedError()
+        messages = [Message(role="user", content=content)]
```

Comment:
this is a good but unrelated change. Ok to move it out of this PR into a new PR?

Reply:
Ha! so funny story ... I just spent a few minutes wondering where my `completion()` code went, because I swore I remembered writing it. I did it in the wrong file! ha. I just had it open as a reference at one point ...

yes, I'll absolutely remove it and put it back in `vllm.py` !

Reply:
removed ... (oops!)